build.gradle

```diff
@@ -1,6 +1,29 @@
-group 'multi-int-store'
-version '0.1.0-SNAPSHOT'
+plugins {
+	id 'org.springframework.boot' version '2.1.4.RELEASE'
+	id 'java'
+}
+
+apply plugin: 'io.spring.dependency-management'
+
+group = 'com.multiintstore'
+version = '0.0.1-SNAPSHOT'
```

Comment:
❗️ 0.1.0-SNAPSHOT

Reply:
I'm not even sure what the proper convention is anymore, but I trust you.

Reply:
[semver](https://semver.org/) mostly cause we started at 0.1.0-snapshot. It seems weird to go backwards

Reply:
We haven't done a release yet, so going backward shouldn't matter, right?

Reply:
@Kjames5269 is right. I see in the FAQ in his link that it's recommended to start development at 0.1.0. They also say we don't have to care too much while we're in development major version 0.